pdoc/doc.py

```diff
@@ -588,15 +608,18 @@ def __public_objs(self):
         """
         _pdoc = getattr(self.module.module, "__pdoc__", {})
 
-        def forced_out(name):
+        def force_skipped(name):
             return _pdoc.get("%s.%s" % (self.name, name), False) is None
 
         def exported(name):
             exported = name == "__init__" or _is_exported(name)
-            return not forced_out(name) and exported
+            inheried = name not in self.cls.__dict__
```

Comment:
This now skips listing inherited members that have not been overridden. E.g.

```py
class C:
   def f(self): pass

class D(C):
   def g(self): pass
```
Here for `D`, only `g()` will be documented, and for `C`, `f`.

The rationale is one can look into linked ancestors for the inherited members, less repetition etc.

Even though it looks quite pretty in the project I'm working with, I imagine not all projects would like/expect this. Can be made a setting if `__pdoc__` "protocol" is extended. Please advise.